examples/index.js

```diff
@@ -2,37 +2,35 @@ import createPhotoPiles from './photos';
 import createMatrixPiles from './matrices';
 import createSvgLinesPiles from './lines';
 import createDrawingPiles from './drawings';
-import createVitessce from './vitessce';
```

Comment:
Same as before: I don't think these deletions are expected. Can you fix these issues manually?